learnosity_sdk/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .request import *
+
+__all__ = [
+        "request"
```

Comment:
There is also `learnosity_sdk.exceptions`. Should that be imported when we `import *`?